.travis.yml

```diff
@@ -26,8 +26,8 @@ deploy:
   on:
     tags: true
     all_branches: true
-    rvm: 1.9.3
-    rvm: 2.0
```

Comment:
Nothing failed, however the required Ruby version in the gemspec for this plugin has already been bumped to 2.1.0 previously so I thought there was little point testing against 2.0.